MOS basics

deskclam opened the pull request.

deskclam committed:
Updated/Added 11 files on 2023-12-19 17:39:01

Affected files:
content/Quantum Devices/MOSFET.md
content/Quantum Devices/attachments/FlatBand.png
content/Quantum Devices/attachments/MOSDepletion.png
content/Quantum Devices/attachments/MOSFETpnJunc.png
content/Quantum Devices/attachments/MOSinversion.png
content/Quantum Devices/attachments/MOSstructure.png
content/Quantum Devices/attachments/MOSth.png
content/Quantum Devices/attachments/NoBias.png
content/Quantum Devices/attachments/Pasted image 20231219100405.png
content/Quantum Devices/attachments/Pasted image 20231219104938.png
content/Quantum Devices/attachments/SurfaceAccumulation.png

deskclam committed:
Merge branch 'gamberoillecito:v4' into v4

gamberoillecito reviewed (changes requested):
add subfolder 01_MOSFET

deskclam committed:
Updated/Added 11 files on 2023-12-19 17:48:32

Affected files:
content/Quantum Devices/01_MOSFET/MOSFET.md
content/Quantum Devices/01_MOSFET/attachments/FlatBand.png
content/Quantum Devices/01_MOSFET/attachments/MOSDepletion.png
content/Quantum Devices/01_MOSFET/attachments/MOSFETpnJunc.png
content/Quantum Devices/01_MOSFET/attachments/MOSinversion.png
content/Quantum Devices/01_MOSFET/attachments/MOSstructure.png
content/Quantum Devices/01_MOSFET/attachments/MOSth.png
content/Quantum Devices/01_MOSFET/attachments/NoBias.png
content/Quantum Devices/01_MOSFET/attachments/Pasted image 20231219100405.png
content/Quantum Devices/01_MOSFET/attachments/Pasted image 20231219104938.png
content/Quantum Devices/01_MOSFET/attachments/SurfaceAccumulation.png

deskclam committed:
Merge remote-tracking branch 'origin/v4' into v4
